modules/programs/kitty.nix

```diff
@@ -79,7 +79,10 @@ in {
       # Generated by Home Manager.
       # See https://sw.kovidgoyal.net/kitty/conf.html
 
-      ${optionalString (cfg.font != null) "font_family ${cfg.font.name}"}
+      ${optionalString (cfg.font != null) ''
+        font_family ${cfg.font.name}
+        font_size ${toString cfg.font.size}
```

Comment:
I would add a null check for name and size, to avoid breaking changes.

Reply:
Do you mean to say that one or both should be nullable? Currently they're mandatory, but font as a whole may be null.

Reply:
Yeah, at least for me both of them should be nullable, or at least the `size` since the `name` was required before. This is to avoid breaking changes,

But I don't know, maybe wait someone that has commit access to comment.

Reply:
It's fair to have it nullable otherwise it will override the programs default for people that dont set the size.

Reply:
Yup, that should work now.